v2/types/value_test.go

```diff
@@ -277,15 +275,13 @@ func TestCloneTimestampPointer(t *testing.T) {
 
 	require.Equal(t, original, cloned)
 	require.NotSame(t, original, cloned)
-	require.NotSame(t, original.Time, cloned.Time)
 }
 
 func TestCloneTime(t *testing.T) {
 	original := time.Now()
 	cloned := types.Clone(original).(types.Timestamp)
 
 	require.Equal(t, original, cloned.Time)
-	require.NotSame(t, original, cloned.Time)
```

Comment:
I don't understand the original... how can it be equal and not equal?

Reply:
I believe it was intended to check whether the values are equal and their addresses differ.

Reply:
that really hurts my head :-)
Sure reads like a bug in "require" to me.
So, in this case did you remove it because "NotSame" fails or because it's redundant?

Reply:
Ah "same" vs "equal". I missed that!

Reply:
Yes! Removed because it seems redundant for me to check that `&original` и `&cloned.Time` are 2 different addresses